docs/codeql/codeql-for-visual-studio-code/setting-up-codeql-in-visual-studio-code.rst

```diff
@@ -64,7 +64,7 @@ There are two ways to do this:
       **Click to show information for LGTM Enterprise users**
 
    Your local version of the CodeQL queries and libraries should match your version of LGTM Enterprise. For example, if you
-   use LGTM Enterprise 1.23, then you should clone the ``1.23.0`` branch of the `starter workspace <https://github.com/github/vscode-codeql-starter/>`__ (or the appropriate ``1.23.x`` branch, corresponding to each maintenance release).
+   use LGTM Enterprise 1.27, then you should clone the ``1.27.0`` branch of the `starter workspace <https://github.com/github/vscode-codeql-starter/>`__ (or the appropriate ``1.27.x`` branch, corresponding to each maintenance release).
```

Comment:
It looks like there exists no `1.27.0` branch for https://github.com/github/vscode-codeql-starter yet, though it will probably be added soon?

Reply:
Thanks for pointing that out, @Marcono1234 👀  There should indeed be a `1.27.0` branch... I've asked about this internally!

Reply:
The `1.27.0` branch has now been created. Thanks again for raising this - much appreciated!